src/client/components/Notifications.tsx

```diff
@@ -39,13 +39,13 @@ export const NotificationAreaPortal = React.createContext<
 >([null, () => {}]);
 
 export function Notifications() {
-  const notifications = entries(useServerState((state) => state.logEntries));
+  const notifications = useServerState((state) => state.logEntries);
   const [lastShownID, setLastShownID] = useState<RRLogEntryID>();
   const [newNotifications, setNewNotifications] = useState<RRLogEntry[]>([]);
   const [hovered, setHovered] = useState(false);
 
   useEffect(() => {
-    const list = notifications;
+    const list = entries(notifications);
```

Comment:
Eventually, we likely want to change the code not use `entries(notifications)` here and in the new code you added, since it unnecessarily needs to create huge arrays when all we really want is to access the last-ish notification. There is probably a cool way in which we could use generators here, but definitely out of scope for this PR.